DemoProject/DemoProject/first_playground.playground/Pages/Advance_Initializairion_de.xcplaygroundpage/Contents.swift

```diff
@@ -493,3 +492,161 @@ for item in arr {
     }
         
 }
+ 
+print()
+//weak unowned
+class Person1 {
+    var name:String
+     var job : Job?
```

Comment:
remove space between : and variable name

Reply:
Okay